README.md

```diff
@@ -954,42 +954,72 @@ Used for EV chargers.
+in   | cmd.state.get_report            | null     |                         | Get the state of the chargepoint, see sup_states
+out  | evt.state.report                | string   |                         | State report of the chargepoint
 -|||
-in   | cmd.cable_lock.set             | bool       |                         | Lock & unlock the cable/connector
-in   | cmd.cable\_lock.get\_report    | null       |                         | Get the status of the cable_lock
-out  | evt.cable_lock.report          | bool       |                         | true = locked, false = unlocked
+in   | cmd.cable_lock.set              | bool     |                         | Lock & unlock the cable/connector
+in   | cmd.cable\_lock.get\_report     | null     |                         | Get the status of the cable_lock
+out  | evt.cable_lock.report           | bool     |                         | true = locked, false = unlocked
 -|||
-in   | cmd.access.set                 | string     |                         | 
-in   | cmd.access.get_report          | null       |                         |
-out  | evt.access.report              | string     |                         |
+in   | cmd.access.set                  | string   |                         | 
+in   | cmd.access.get_report           | null     |                         |
+out  | evt.access.report               | string   |                         |
 -|||
-in   | cmd.smart_charge.set           | bool       |                         | true = enable smart charging
-in   | cmd.smart_charge.get_report    | null       |                         | 
-out  | evt.smart_charge.report        | bool       |                         |
+in   | cmd.smart_charge.set            | bool     |                         | true = enable smart charging
+in   | cmd.smart_charge.get_report     | null     |                         | 
+out  | evt.smart_charge.report         | bool     |                         |
 -|||
-in   | cmd.rated_current.get_report   | null       |                         | Get the rated or max current for ths CP
-out  | evt.rated_current.report       | int        |                         |
+in   | cmd.rated_current.get_report    | null     |                         | Get the rated or max current for ths CP
+out  | evt.rated_current.report        | int      |                         |
 -|||
-in   | cmd.current_session.get_report | null       |                         | Command for getting energy (kWh) for current session.
-out  | evt.current_session.report     | float      |                         |
+in   | cmd.current_session.get_report  | null     |                         | Command for getting energy (kWh) for current session.
+out  | evt.current_session.report      | float    |                         |
 -|||
-out  | evt.error.report               | string     |                         | 
+in   | cmd.electric_current.set        | object   |                         | Command for setting an electric current.
```

Comment:
Put the object example in this section instead of Examples at the end. See this example for reference:
![image](https://user-images.githubusercontent.com/31538120/167607137-eab96923-9f7b-4da8-b14d-722997b7e074.png)

Reply:
Fixed.